utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/DebuggerController.java

```diff
@@ -10,6 +10,7 @@
 // Dummy line
 // Dummy line
 // Dummy line
+// Dummy line
```

Comment:
why?

Reply:
The `debugger` feature allows attachment to specific lines of code. Due to differences in line numbering between `dotnet` and `java`, 'dummy lines' are used to standardize this functionality.

I agree with you, leaving just `dummy lines` without any explanation was not clear. I have added explanations in the comments here: 22f223d76dfa3570794c64f57d1a35b48d2b7c1b